scripts/miscContracts.ts

```diff
@@ -98,7 +98,7 @@ export async function getMiscAddresses() {
 		data: {
 			name,
 			protocol,
-			logoURI: undefined,
+			logoURI: null,
```

Comment:
I think `undefined` is more correct here.  Alternatively you could do this:
```suggestion
			...logoURI ? { logoURI } : {},
```